apps/haproxy/haproxy.cfg

```diff
@@ -33,14 +33,18 @@ acl host_prometheus_backend hdr_dom(host) -i prometheus.ly.fish.local
 use_backend prometheus_backend if host_prometheus_backend
 
 frontend tcp_5432_in
-mode tcp
-bind *:5432
-default_backend postgres_backend
+  bind *:5432
+  mode tcp
+  log global
+  timeout client 1440m
```

Comment:
I would rather set timeout settings in a global way and not on a per-service basis. This is done in this file in the `defaults` section, and in a slightly more legible way here: https://github.com/product-os/jellyfish/pull/6847/files#diff-ae1fba26d587e6e571217dbbd34ecdaca8b9adc59423ef18c144193debf30995

Maybe we can bring timeout related settings from that PR into this one and tweak the timeouts to be a bit more if necessary. Also, is there a reason you went with 1440 minutes? It seems a bit excessive. There are downsides to turning this up too much: https://discourse.haproxy.org/t/turn-off-all-timeouts/91

Reply:
We need to remember that haproxy will be used in production before too long and not assume that these settings are only for development.